README.md

```diff
@@ -33,20 +33,21 @@ GPflow would not be the same without the community. **We are grateful to [all co
 
 GPflow implements modern Gaussian process inference for composable kernels and likelihoods. The [online documentation (develop)](http://gpflow.readthedocs.io/en/develop/)/[(master)](http://gpflow.readthedocs.io/en/master/) contains more details.
 
-GPflow 2.0 uses [TensorFlow 2.1+](http://www.tensorflow.org) for running computations, which allows fast execution on GPUs, and uses Python ≥ 3.6.
-
+GPflow 2.1 uses [TensorFlow 2.2+](http://www.tensorflow.org) for running computations, which allows fast execution on GPUs, and uses Python ≥ 3.6.
```

Comment:
gpflow 2.1.3 allows tf 2.1

Reply:
It doesn't nominally require tf 2.2 itself, but it does require tfp 0.10+ which in turn only works with tf 2.2 or larger...

Reply:
aha. ok cool